app/models/main.py

```diff
@@ -529,6 +534,12 @@ class SupplierFramework(db.Model):
         nullable=True,
     )
 
+    # Suppliers must confirm that their details are accurate for every framework application they make.
+    # The flag `company_details_confirmed` is set on the Supplier object the first time a supplier confirms their
+    # details (company name+number, vat number, duns number). The below SupplierFramework flag is set when a
+    # supplier confirms their details with an open application to an iteration of a framework.
+    application_company_details_confirmed = db.Column(db.Boolean, index=False, default=False, nullable=True)
```

Comment:
Do you need to specify `False` for index? Isn't it the default?
Is `default=False` the kind of default that isn't going to lock everything when the migration is run?

Reply:
Don't need to specify it, but explicit is better than implicit 🤷‍♂️ 

`default=False` won't lock the table here as we're setting it to nullable. This just means in the future it will default to False, but it won't touch existing rows.